### ISSUE TITLE
Strange bug related to import_map.json

### ISSUE THREAD

**Issue by al6x** [denoland/vscode_deno#422]
VS Code Deno can't find the method on the class. It is a valid code and those methods exists and code compiled correctly by deno.

<img width="731" alt="Screen Shot 2021-05-30 at 2 37 49 pm" src="https://user-images.githubusercontent.com/94196/120102658-b7ccd480-c154-11eb-8861-b8c1f70ccf5b.png">

It's hard to create an isolated test case for it, so I wrote a screen recording showing how it looks like https://youtu.be/Ql3JgfMGoUM

It **seems it's related to import_map.json**, as when I use the absolute paths in imports it works correctly.

**Versions**

vscode: 1.56.2 extension: v3.4.0

deno 

```
deno 1.10.2 (release, x86_64-apple-darwin)
v8 9.1.269.27
typescript 4.2.2
```

**Comment by jeiea** [comment]
Did you set `deno.importMap` by settings.json? I can't find it from your youtube clip.
I think reproducible repository is more appropriate to example for bug report.

**Comment by al6x** [comment]
The content of the `settings.json`, the path to import_map is correct, as sometimes VS Code works well. Maybe it's caused by relative path, I'll try to update to absolute and see if problem go away, and will create sample repo a bit later.

```Nim
{
  "deno.enable": true,
  "deno.unstable": true,
  "deno.importMap": "./import_map.json"
}
```

**Comment by justinmchase** [comment]
The docs say to use `"deno.importMap"` but the plugin appears to be actually expecting `"deno.import_map"`. When I change it to that it works fine. So it seems like either the plugin or the documentation is off.

![Screen Shot 2021-07-02 at 12 23 16 PM](https://user-images.githubusercontent.com/10974/124309525-7606c380-db30-11eb-828d-2c2a93b8a85e.png)

![Screen Shot 2021-07-02 at 12 23 28 PM](https://user-images.githubusercontent.com/10974/124309529-7737f080-db30-11eb-8e67-d79d6c5c59d6.png)

![Screen Shot 2021-07-02 at 12 24 01 PM](https://user-images.githubusercontent.com/10974/124309534-78691d80-db30-11eb-813b-be05980a8a39.png)

**Comment by jeiea** [comment]
![image](https://user-images.githubusercontent.com/4188984/124311431-b73aba00-dba8-11eb-841b-fe9f08754975.png)
It seems you are using old extension @justinmchase.

**Comment by justinmchase** [comment]
Derp. I guess I had a pending update that was preventing me from getting the latest here. I must have cancelled it in the past and it was waiting on me to apply it.

**Comment by sharifzadesina** [comment]
Same here and I am using the latest extension version.
The extension works fine with `deno.import_map` but inside the settings, It suggests `deno.importMap`.
I think you need to update this to use `deno.importMap`.

**Comment by kitsonk** [comment]
@sharifzadesina version of the extension and the output of "Deno Language Server" from vscode please.  It shouldn't be doing what you are suggesting it is doing, as @justinmchase noted, he had an outdated version of the extension.

**Comment by kitsonk** [comment]
People had various issues here, and not sure if there is anything specifically actionable.

There are some issues when using import maps and types, which is being covered under denoland/deno#11456.

Outside of that, `import_map` _may_ work with a modern version of the extension, as the language server deserialises the JSON in Rust, and it just happens that `importMap` gets deserialized to `import_map` inside the language server.  It is accidental, but for version 3.X of the extension, the setting is `importMap` and the only _supported_ setting.

Because everything else is not clear, if it isn't denoland/deno#11456 please open a new issue.
